languages/csharp/exercises/concept/classes/.docs/instructions.md

```diff
@@ -0,0 +1,81 @@
+In this exercise you'll be playing around with a remote controlled car, which you've finally saved enough money for to buy.
+
+Cars start with full (100%) batteries. Each time you drive the car using the remote control, it covers 20 meters and drains one percent of the battery.
+
+The remote controlled car has a fancy LED display that shows two bits of information:
+
+- The total distance it has driven, displayed as: `"Driven <METERS> meters"`.
+- The remaining battery charge, displayed as: `"Battery at <PERCENTAGE>%"`.
+
+If the battery is at 0%, you can't drive the car anymore and the battery display will show `"Battery empty"`.
+
+You have six tasks, each of which will work with remote controller car instances.
+
+### 1. Buy a brand-new remote controller car
+
+Implement the (_static_) `Buy()` method to return a brand-new remote controlled car instance:
```

Comment:
I'm also unsure if I like omitting the class name. This will be a bit more problematic if there are multiple classes. Another option would be to do:

```suggestion
Implement the (_static_) `RemoteControlCar.Buy()` method to return a brand-new remote controlled car instance:
```

Reply:
Or:

```suggestion
Implement the (_static_) `Buy()` method on the `RemoteControlCar` class to return a brand-new remote controlled car instance:
```

Reply:
I'm ok with `Implement the (_static_) `RemoteControlCar.Buy()` method`.

Reply:
Great